OMX-27.ino

```diff
@@ -191,33 +190,35 @@ void readPotentimeters(){
    	
     	if(analog[k]->hasChanged()) {
        		// do stuff		   	
-			switch(mode) { 
-				case 3:
+			switch(omxMode) { 
+				case MODE_OM:
 						// fall through - same as MIDI
-				case 0: // MIDI
-					sendPots(k);
+				case MODE_MIDI: // MIDI
+					sendPots(k, midiChannel);
 					dirtyDisplay = true;
 					break;    	
 
-				case 2: // SEQ2
+				case MODE_S2: // SEQ2
 						// fall through - same as SEQ1
-
-				case 1: // SEQ1
+				case MODE_S1: // SEQ1
 					if (noteSelect && noteSelection){ // note selection - do P-Locks
 						potNum = k;
 						potCC = pots[k];
 						potVal = analogValues[k];
 								// stepNoteP[8] {notenum,vel,len,p1,p2,p3,p4,p5}
 						if (k < 4){ // only store p-lock value for first 4 knobs
-							stepNoteP[playingPattern][selectedStep][k+3] = analogValues[k];
+							stepNoteP[playing][selectedStep].params[k] = analogValues[k];
```

Comment:
Bug? `playing` is a global for transport. 

This should be `playingPattern` perhaps?

Reply:
Yup think you're right